domains/novafox.json

```diff
@@ -4,6 +4,8 @@
         "email": "foxsdenyt@gmail.com"
     },
     "record": {
-        "CNAME": "foxsdenyt-github-io.onrender.com"
+        "A": "216.24.57.1",
```

Comment:
record should be an array of IPs

Reply:
Also i cannot reach your website
error 1003 Direct IP access not allowed

Reply:
oh... do u know if i can use alias records? Render's docs say alias can be used to coexist with other records

Reply:
It's set as Alias now, :crossed_fingers: hopefully this'll work

Reply:
the checker is failing over an older commit smh